### PR TITLE
Merge Feature Tours Pull Request

### PR DESCRIPTION
**Pull request by ngocquyhoang**
Completed Feature Tours Pull Request

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -13,7 +13,8 @@ gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.1.0'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
-
+# add font awesome
+gem "font-awesome-rails"
```
Comment (tuanha) [inline Gemfile]: remove the gem

File: app/controllers/tours_controller.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      if @tour.update(tour_params)
+        format.html { redirect_to @tour, notice: 'Tour was successfully updated.' }
+      else
+        format.html { render :edit }
+      end
+    end
+  end
+
+  def destroy
+    @tour.destroy
```
Comment (tuanha) [inline app/controllers/tours_controller.rb]: khong dung bien @ neu ko su dung ngoai view

File: app/controllers/tours_controller.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    @tour = Tour.new
+  end
+
+  def edit
+  end
+
+  def create
+    @tour = Tour.new(tour_params)
+    @tour.company_id = current_user.company.id
+
```
Comment (tuanha) [inline app/controllers/tours_controller.rb]: khong dung bien @ neu ko su dung ngoai view

File: app/views/tours/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+<style type="text/css">
+  .list-tour tr[visible='false'],
+  .no-result{
+    display:none;
+  }
+
+  .list-tour tr[visible='true']{
+    display:table-row;
+  }
+</style>
```
Comment (tuanha) [inline app/views/tours/index.html.erb]: css dua vao file .css

**Comment by tuanha** [comment]
chinh lai giao dien trang tour/index
lam cho dong bo vs trang devices nua

File: app/controllers/tours_controller.rb (hunk 3 above)
Comment (ngocquyhoang) [inline app/controllers/tours_controller.rb]: cai nay su dung nhe, con return error ra view

File: app/assets/stylesheets/application.css [hunk 5]
```diff
@@ -14,4 +14,5 @@
  *= require devise_bootstrap_views
  *= require_tree .
  *= require_self
+ *= require font-awesome
```
Comment (tuanha) [inline app/assets/stylesheets/application.css]: remove no di, bi trung

**Pull request closed without merging**